### PR TITLE
First version of Mgmt summary

### PR DESCRIPTION
**Pull request by lstreckeisen**

### CODE REVIEW

File: SA/design_concept/mgmt_summary.typ [hunk 1]
```diff
@@ -1,8 +1,34 @@
 = Management Summary
+// TODO: finalize
 
-Content:
-- initial situation / problem description
-- procedure of finding solution
-- results & outlook
+When introducing children and young adults to the world of programming, many teachers use tools like Scratch or LEGO Mindstorms.
+Such visual, block-based tools eliminate the hurdle of code syntax, allowing beginners to concentrate on the program they want to write.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:5]:
```suggestion
Such visual, block-based tools eliminate the hurdle of code syntax, allowing beginners to concentrate on the program they want to write.

```

File: SA/design_concept/mgmt_summary.typ [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+However, all popular visual tools used for teaching children are made for imperative programming.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:6]:
```suggestion
However, all popular visual tools used for teaching programming are made for imperative programming.
```

File: SA/design_concept/mgmt_summary.typ [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+There exist visual tools for functional programming, but they lack in visual aestethics or hide important concepts that are imperative for understanding the functional programming paradigm.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:7]:
```suggestion
There exist visual tools for functional programming, but they lack in visual aesthetics or hide important concepts that are vital to understanding the functional programming paradigm.
```

File: SA/design_concept/mgmt_summary.typ [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
+
+With VisualFP, a visual, block-based tool should be designed that can be used to teach functional programming.
+At the center of this project is a design concept for visual function composition, describing how the visual editor of such a tool should work.
+To proof that the theoretical concept is feasable, a proof of concept application with a visual function editor should be created.
+
+To find a suitable design for the function editor, two design iterations were carried out.
+After creating 3 initial design proposals during the first iteration, a small survey with experienced developers & students was conducted to gain insight into the advantages & disadvantages of each design.
+Based on this feedback a fourth, more detailed, design concept was created.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:15]:
```suggestion
After creating 3 initial design proposals during the first iteration, a small survey with experienced developers & students was conducted to gain insight into the advantages & disadvantages of each design.
Based on this feedback, a fourth, more detailed design concept was created.
```

File: SA/design_concept/mgmt_summary.typ [hunk 5]
```diff
@@ -1,8 +1,34 @@
...
+
+This concepts uses nested blocks to represent function definitions.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:17]:
```suggestion
This concept uses nested blocks to represent function definitions.
```

File: SA/design_concept/mgmt_summary.typ [hunk 6]
```diff
@@ -1,8 +1,34 @@
...
+At the center of the concept are type holes whose type is determined automatically based on the already defined parts of a function.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:18]:
```suggestion
The development flow is guided by type holes, which are typed placeholders users can drop blocks into. The placeholders are added by the application as the values required for the definition to be valid.
```

I am very unsure if this sounds right

File: SA/design_concept/mgmt_summary.typ [hunk 7]
```diff
@@ -1,8 +1,34 @@
...
+Like this the editor guides the user during function composition allowing them to concentrate on the function values they want to use.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:19]:
```suggestion
```

File: SA/design_concept/mgmt_summary.typ [hunk 8]
```diff
@@ -1,8 +1,34 @@
...
+The application includes a small selection of pre-defined values that can be used in the function editor to build a user-defined function.
+
+#figure(
+  image("./static/visualFP_mapAdd5.png", width: 18cm),
+  caption: "Screenshot of the mapAdd5 function definition in VisualFP"
+)
+
+The implemented application proofs that the developed design concept for the function editor works as envisioned.
+However, the application is not yet ready to be used for teaching.
+For that, an additional project would have to be carried out that creates a visual editor for custom data types and allows the execution of defined functions.
```
Comment (fl3pp) [inline SA/design_concept/mgmt_summary.typ:32]:
```suggestion
However, the application is not yet ready to be used in classrooms.  It is merely a proof, additional design and development is required in order to bring the concept to its full potential.
```

File: SA/design_concept/mgmt_summary.typ (hunk 6 above)
Comment (lstreckeisen) [reply SA/design_concept/mgmt_summary.typ]: I reworded it a bit

**Review by fl3pp — APPROVED** [review]

**Pull request merged**